lightning/src/ln/features.rs

```diff
@@ -39,8 +39,13 @@
 //!     (see [BOLT-4](https://github.com/lightning/bolts/blob/master/04-onion-routing.md) for more information).
 //! - `BasicMPP` - requires/supports that a node can receive basic multi-part payments
 //!     (see [BOLT-4](https://github.com/lightning/bolts/blob/master/04-onion-routing.md#basic-multi-part-payments) for more information).
+//! - `Wumbo` - requires/supports that a node create large channels. Called `option_support_large_channel` in the spec.
```

Comment:
Could say "large channels (superior to `MAX_FUNDING_SATOSHIS_NO_WUMBO`)"

Reply:
Ah, that const isn't public